sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -170,4 +172,33 @@ export class ApplicationService {
   async reissueMSFAA(applicationId: number): Promise<void> {
     await ApiClient.Application.reissueMSFAA(applicationId);
   }
+
+  /**
+   * Triggers a manual reassessment for the application.
+   * @param applicationId application id.
+   * @param payload contains notes for the application.
+   * @returns id of the assessment just created.
+   */
+  async triggerManualReassessment(
+    applicationId: number,
+    payload: ManualReassessmentAPIInDTO,
+  ): Promise<PrimaryIdentifierAPIOutDTO> {
+    return ApiClient.Application.triggerManualReassessment(
+      applicationId,
+      payload,
+    );
+  }
+
+  /**
+   * Gets application and assessment status details.
+   * @param applicationId application id.
+   * @returns application and assessment details.
+   */
+  async getApplicationAssessmentStatusDetails(
```

Comment:
This method was moved from controller in the API and probably need be moved in the Web also, right?

Reply:
That one is in the application service. I moved `triggerManualReassessment` to the assessment service